earthmover/__main__.py

```diff
@@ -99,10 +99,15 @@ def main(argv=None):
         action='store_true',
         help='overwrites `show_stacktrace` config in the config file to true; sets `log_level` to DEBUG'
     )
-    parser.add_argument("--results-file",
+    parser.add_argument("-r", "--results-file",
```

Comment:
I added this (why not? 😄 other flags have a short version, and `-r` is available)